src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -1038,6 +1038,11 @@ protected void ExecuteIndexing()
                                     if (_logger.IsInfoEnabled)
                                         _logger.Info($"Finished indexing for '{Name}'.'");
                                 }
+                                catch (TimeoutException te)
+                                {
+                                    if (_logger.IsOperationsEnabled)
```

Comment:
That can happen? Under what scenario?

Reply:
ping

Reply:
This can happen when we very slow IO.
Index flushing is in progress and we try to open a write tx.